fontforge/ufo.c

```diff
@@ -80,23 +80,23 @@ static char *buildname(const char *basedir, const char *sub) {
 return( fname );
 }
 
-static void extractNumericVersion(const char * textVersion, int * versionMajor, int * versionMinor) {
+static int extractNumericVersion(const char * textVersion, int * versionMajor, int * versionMinor) {
   // We extract integer values for major and minor versions from a version string.
   *versionMajor = -1; *versionMinor = -1;
-  if (textVersion == NULL) return;
+  if (textVersion == NULL) return 0;
   char *nextText1 = NULL;
   char *nextText2 = NULL;
   int tempVersion = -1;
   tempVersion = strtol(textVersion, &nextText1, 10);
   if (tempVersion != -1 && nextText1 != NULL && (nextText1[0] == '\0' || nextText1[0] == ' ' || nextText1[0] == '.')) {
     *versionMajor = tempVersion;
-  } else return;
-  if (nextText1[0] == '\0') return;
+  } else return 0;
+  if (nextText1[0] == '\0') return 1;
   tempVersion = strtol(nextText1+1, &nextText2, 10);
   if (tempVersion != -1 && nextText2 != NULL && (nextText2[0] == '\0' || nextText2[0] == ' ' || nextText2[0] == '.')) {
     *versionMinor = tempVersion;
-  } else return;
-  return;
+  } else return 1;
+  return 2;
```

Comment:
I think that it is unnecessary and duplicative to add a set of return flags that just mirror the values placed in versionMajor and versionMinor.